README.md

```diff
@@ -56,11 +56,59 @@ The structure should appear like so:
 ├── documentation
 └── readme.md
 ```
+## New Directory structure (effective July 10th 2017)
 
+Re-organization is being done to support upcoming profiles work and Data Plane swagger specs. The new structure will appear like below
+ 
++---automation
+|   \---resource-manager
+|       \---Microsoft.Automation
+|           \---2015-10-31
+|               \---examples
++---batch
+|   +---data-plane
+|   |   \---Microsoft.Batch
+|   |       +---2015-12-01.2.2
+|   |       +---2016-02-01.3.0
+|   |       +---2016-07-01.3.1
+|   |       +---2017-01-01.4.0
+|   |       |   \---examples
+|   |       \---2017-05-01.5.0
+|   \---resource-manager
+|       \---Microsoft.Batch
+|           +---2015-12-01
+|           +---2017-01-01
+|           |   \---examples
+|           \---2017-05-01
+|               \---examples
++---billing
+|   \---resource-manager
+|       \---Microsoft.Billing
+|           +---2017-02-27-preview
+|           |   \---examples
+|           \---2017-04-24-preview
+|               \---examples
+└── readme.md
+ 
+### Changing the representation of composite swagger metadata
+
+Currently composite swagger metadata is captured in a .json file. The problem with the current representation is that it does not provide a way to track what versions of swagger specs were used in earlier releases. With the upcoming changes, we will be able to keep track of multiple versions of composition. The metadata will be captured in readme.md file under resource-manager folder. There won’t be a need for .json file. New structure in readme.md file looks like below. 
+Tag: 2017-03-30
+These settings apply only when --tag=2017-03-30 is specified on the command line.
+input-file:
+- Microsoft.Compute/2017-03-30/compute.json
+- Microsoft.Compute/2017-03-30/disk.json
+- Microsoft.Compute/2017-03-30/runCommands.json
+- Microsoft.ContainerService/2017-01-31/containerService.json
+
+Please take a look at this [readme.md](https://github.com/Azure/azure-rest-api-specs/tree/reorg/specification/compute/resource-manager) to understand how the complete structure looks like.
+
+
+## Specs Format
 Currently, the specifications are expected to be in Swagger JSON format
 
 ## Next steps
 The next step in the process after a spec is completed is to generate SDKs and API reference documentation. Go to the [Azure Developer Experience guide](https://github.com/Azure/adx-documentation-pr) for more information.
```

Comment:
It looks like we provided this info at the top of this page. It does not hurt to say it again but I will let you decide that.